### DISCUSSION TITLE
Manual Backups

### DISCUSSION THREAD

**Discussion by p-karanthaker** [lloesche/valheim-server-docker#232] [Q&A] [answered]
Hi, is there any way to trigger a backup manually?

**Comment by lloesche** [comment, marked as answer]
Yes, sending `SIGHUP` to the `valheim-backup` service or restarting the service will create a backup.
The PID of the running service can be found in `/var/run/valheim-backup.pid`

Assuming your container's name is `valheim-server` here's how both would work:

Sending SIGHUP using `supervisorctl` (the most graceful way of making a backup)
```
docker exec -it valheim-server supervisorctl signal HUP valheim-backup
```

Sending SIGHUP manually (as graceful as before but more "manual" work)
```
docker exec -it valheim-server bash -c 'kill -HUP $(< /var/run/valheim-backup.pid)'
```

Restarting `valheim-backup` (the more brute force way)
```
docker exec -it valheim-server supervisorctl restart valheim-backup
```

The restart can also be done from [the supervisor web UI](https://github.com/lloesche/valheim-server-docker#supervisor).
![image](https://user-images.githubusercontent.com/2124094/111460961-eb36c000-871c-11eb-8233-15c6db441e26.png)

**Reply by p-karanthaker** [reply]
Amazing, this is really helpful! Thanks!
